services/arrow-flight/src/main/java/com/dremio/service/flight/impl/FlightPreparedStatement.java

```diff
@@ -68,6 +68,18 @@ public FlightInfo getFlightInfo(Location location) {
     return new FlightInfo(schema, flightDescriptor, ImmutableList.of(flightEndpoint), -1, -1);
   }
 
+  public ActionCreatePreparedStatementResult createAction() {
```

Comment:
JavaDocs please. More importantly, why is this public on the user-facing FlightPreparedStatement?

Reply:
Ah - I see, it's not in the flight client, which is good. But still, please add JavaDocs. Or revert this change since I think it's just a method move.

Reply:
The method moved during the rebases. I added the java doc